src/react/useAtom.ts

```diff
@@ -32,11 +32,6 @@ export function useAtom<
   SetAtom<ExtractAtomArgs<AtomType>, ExtractAtomResult<AtomType>>
 ]
 
-export function useAtom<AtomType extends Atom<unknown>>(
```

Comment:
I wonder if `unknown` is problematic. Can you try `any` instead? If that works, it was my bad then...